source/stylesheets/_default.scss

```diff
@@ -695,3 +695,17 @@ $screen-minimum: 320px;
 	-webkit-focus-ring-color:  rgba(0, 0, 0, 0);
 	outline: none;
 }
+
+@mixin icon($color, $shade-color: null) {
+	circle {
+		@include transition(fill $def-transition_duration);
+
+		fill: $color;
+
+		@if $shade-color {
+			&:hover {
+				fill: $def-danger-shade;
+			}
+		}
+	}
+}
```

Comment:
Define the mixin locally, inside the `.icon` selector: the *_default.scss* is reserved for general mixins, whereas this is very specific for the below 2 cases.

Reply:
moved out `circle` part so made it more reusable